### PR TITLE
[Promscale] Continue adding frontmatter

### PR DESCRIPTION
**Pull request by charislam**
# Description

Keep moving excerpts, keywords, and tags to frontmatter

# Links

Partial #1281 

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Are procedure and highlight tags used appropriately?
- [ ] Has the index been updated appropriately?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] Are all links provided in reference style, and resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: promscale/visualize-data/apm-experience.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+---
+title: Application Performance Management (APM) with traces
+excerpt: APM experience within Grafana using dashboards with SQL queries on traces
+product: promscale
+keywords: [Jaeger, APM]
+tags: [configure, opentelemetry, traces]
+---
+
 # Application Performance Management (APM) with traces
```
[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply promscale/visualize-data/apm-experience.md:9]: That is right @Loquacity !!! We need to change all the instances to Application performance monitoring.

[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply promscale/visualize-data/apm-experience.md]: Just committed the changes. Let me know if the changes are fine.

**Comment by Rajakavitha1** [comment]
This is lot of work @charislam !!!! Thank you for the PR.

File: promscale/visualize-data/apm-experience.md [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
 keywords: [Jaeger, APM]
 tags: [configure, opentelemetry, traces]
 ---
 
-# Application Performance Management (APM) with traces
-Promscale provides an APM-like experience with traces data using SQL. Import the 
-Grafana dashboards that are published by the Promscale team to get the APM 
-with traces.
+# Application performance monitoring (APM) with traces
+Promscale provides application performace monitoring with traces data using SQL. Import the
```
Comment (Loquacity) [inline promscale/visualize-data/apm-experience.md:10]:
```suggestion
Promscale provides application performance monitoring with traces data using SQL. Import the 
```

File: promscale/visualize-data/apm-experience.md [hunk 3]
```diff
@@ -63,10 +62,10 @@ dashboards published by Promscale.
 1.  In  the `Folder` drop-down menu, choose the folder to which you want to add
     the dashboard.
 1.  Select the data sources from which you want the dashboard to query the data:
-    * For APM dashboards select `TimescaleDB or PostgreSQL data source` as
-      `Promscale-SQL`.
-    * For APM dashboards select `Promscale Jaeger Tracing data source` as
-      `Promscale-Tracing`.
+    * For application performance monitoring dashboards select `TimescaleDB or PostgreSQL
+      data source` as `Promscale-SQL`.
+    * For application performance monitoringdashboards select `Promscale Jaeger Tracing data source`
```
Comment (Loquacity) [inline promscale/visualize-data/apm-experience.md:67]:
```suggestion
    * For application performance monitoring dashboards, select `Promscale Jaeger Tracing data source`
```

**Review by Rajakavitha1 — APPROVED** [review]

File: promscale/visualize-data/apm-experience.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Loquacity) [reply promscale/visualize-data/apm-experience.md]: LGTM 👍

File: promscale/visualize-data/apm-experience.md (hunk 3 above)
Comment (charislam) [inline promscale/visualize-data/apm-experience.md]:
```suggestion
    * For application performance monitoring dashboards, select `Promscale Jaeger Tracing data source`
```

File: promscale/visualize-data/apm-experience.md [hunk 4]
```diff
@@ -88,10 +87,10 @@ dashboards published by Promscale.
 1.  In  the `Folder` drop-down menu, choose the folder to which you want to add
     the dashboard.
 1.  Select the data sources from which you want the dashboard to query the data:
-    * For APM dashboards select `TimescaleDB or PostgreSQL data source` as
-      `Promscale-SQL`.
-    * For APM dashboards select `Promscale Jaeger Tracing data source` as
-      `Promscale-Tracing`.
+    * For application performance monitoring dashboards select `TimescaleDB or PostgreSQL data source`
```
Comment (charislam) [inline promscale/visualize-data/apm-experience.md:90]:
```suggestion
    * For application performance monitoring dashboards, select `TimescaleDB or PostgreSQL data source`
```

File: promscale/visualize-data/apm-experience.md [hunk 5]
```diff
@@ -88,10 +87,10 @@ dashboards published by Promscale.
...
+      as `Promscale-SQL`.
+    * For application performance monitoring dashboards select `Promscale Jaeger Tracing data source`
```
Comment (charislam) [inline promscale/visualize-data/apm-experience.md:92]:
```suggestion
    * For application performance monitoring dashboards, select `Promscale Jaeger Tracing data source`
```

**Pull request merged**